js/base/Exchange.js

```diff
@@ -2583,7 +2583,7 @@ module.exports = class Exchange {
                 return markets[0];
             }
         }
```

Comment:
```suggestion
                return markets[0];
            } else {
                throw new BadSymbol (this.id + ' does not have market symbol ' + symbol);
            }
        }
```

Reply:
tbh, I still havent got an answer to above my comment, and I dont see any difference if we put this in `else` clause or not. my PR tried to expose completely different point.
however, if you want, I'll commit what you say.